src/test/resources/application-integration-test.yml

```diff
@@ -8,6 +8,6 @@ workspace.integration-test:
   wsm-endpoints:
     workspaces: api/workspaces/v1
   wsm-urls:
-# swap the commented line to run integration tests against local
-    dev: https://workspace.dsde-dev.broadinstitute.org/
-#    dev: http://localhost:8080/
+# swap the commented line to run integration tests against dev
+#    dev: https://workspace.dsde-dev.broadinstitute.org/
+    dev: http://localhost:8080/
```

Comment:
It feels kinda hacky to point at localhost but still call it dev. 

The slightly-less hacky solution would be changing the value of TEST_ENV in the github action, but that also requires copying values for `data-repo-instance-names` and `data-repo-snapshot-id`. I'm not sure that's worth it for a temporary solution like this

Reply:
I'm not that big a fan of TEST_ENV at all. I'd prefer just configuring the values instead of configuring a map of envs to values and then configuring then env. e.g. `data-repo-instance-name: terra`. We can have multiple application properties that we include on the fly if it's really interesting to run these tests against non dev environments.
As-is, what does it mean for there to be a local TEST_ENV? there's not a local DR to refer to, so we end up duplicating values. WDYT?

Reply:
Yeah, that's the doubt I had. Given that it's ultimately only changing the wsm-url and this will be fairly short lived, I'm fine with the change as is.